.azure-pipelines/templates/test.yml

```diff
@@ -9,7 +9,7 @@ parameters:
 
 jobs:
   - ${{ each job in parameters.jobs }}:
-    - job: test_${{ replace(replace(replace(job.test, '/', '_'), '.', '_'), '-', '_') }}
+    - job: test_${{ replace(replace(replace(replace(job.test, '/', '_'), '.', '_'), '-', '_'), '@', '_') }}
```

Comment:
Is there some central repository you need to update for this. I believe @mattclay has some script that auto generates these files from a template.

Reply:
That file is here: https://github.com/ansible/shippable-migration-tool/blob/main/content/templates/test.yml

It's only used for migrating from Shippable to AZP. We don't have an automated process in place yet for keeping all the templates in sync on AZP, but once we do, it should probably be in the [ansible-azp-tools](https://github.com/ansible/ansible-azp-tools) repo.

Reply:
I think there's a tool that generates similar things from templates for collection repos, but IIUC these are hand-rolled. 

This was a short-term hack made at @mattclay's suggestion to get non-native Python selection baked into the job IDs without ansible-test needing to be made directly aware of them. I think he's got a longer-term plan for this that would allow it to be supported everywhere, but we needed a quick way to test on > native system Python- I'll let him add any detail if he wants...